sdk/keyvault/Azure.Security.KeyVault.Keys/tests/LocalCryptographyProviderFactoryTests.cs

```diff
@@ -47,6 +47,18 @@ public void NoKeyMaterial()
             Assert.IsNull(provider);
         }
 
+        [Test]
+        public void NoOctKeyMaterial()
```

Comment:
Maybe name it something like this?
```suggestion
        public void ProviderIsNotInitializedWhenOctKeyTypeHasNoPrivateKey()
```

Reply:
The name is a variation on the one above it, that basically tests the same thing against a different code path. If I have any other changes, I'll consider it. I tend not to go with super long names because they always get effectively truncated in Test Explorer, Azure Pipelines, etc., anyway.